src/juju/api.ts

```diff
@@ -774,16 +774,17 @@ export async function getCharmsURLFromApplications(
   return charms.filter((charm) => !!charm).map((charm) => charm?.url);
 }
 
-export function findCrossModelQuery(
+export function crossModelQuery(
```

Comment:
I left the name as `crossModelQuery` as I couldn't really think of anything better. If you have some proposals, don't hesitate to tell them! 🙂

Reply:
I think that's fine!